controllers/shipwrightbuild_controller.go

```diff
@@ -131,13 +149,29 @@ func (r *ShipwrightBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	logger.Info("Applying manifest's resources...")
 	if err := manifest.Apply(); err != nil {
 		logger.Error(err, "Rolling out manifest's resources")
+		apimeta.SetStatusCondition(&b.Status.Conditions, metav1.Condition{
+			Type:    ConditionReady,
+			Status:  metav1.ConditionFalse,
+			Reason:  "Failed",
+			Message: "Reconciling ShipwrightBuild failed",
```

Comment:
Message should also include the error:
```suggestion
			Message: fmt.Sprintf("Reconciling ShipwrightBuild failed: %v", err)
```

Reply:
done